dune

```diff
@@ -1,6 +1,4 @@
 (rule
- (target graphql_schema.json)
+ (targets graphql_schema.json)
```

Comment:
Unrelated change, please remove. I also feel that using `%{target}` is more natural, but we can discuss on the dedicated PR if you still want to make this change.

Reply:
Good catch! Looks like a rebase mistake to me (I changed from `targets` to `target` just recently in another PR).